lib/crypto/crypto_compat.c

```diff
@@ -213,3 +213,18 @@ crypto_compat_RSA_generate_key()
 	/* Failure! */
 	return (NULL);
 }
+
+/**
+ * crypto_compat_free(void):
+ * Free the shared memory allocated by OpenSSL.
+ */
+void
+crypto_compat_free(void)
```

Comment:
I'm not certain if you'd prefer `crypto_compat_cleanup()` here instead.  Most (if not all) of our other `_free` functions take a structure, but I figured that since this was a `(void)` it would be sufficiently clear.